### PR TITLE
Ut for getCustomisation razorpay-affordability-widget.php

### PR DESCRIPTION
**Pull request by garimaprachi**

### CODE REVIEW

File: tests/phpunit/tests/test-affordabilitywidget.php [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+        $response = getFooterFontSize();
+
+        $this->assertSame('10', $response);
+    }
+
+    public function testgetCustomisation()
+    {
+        add_option('rzp_afd_color', 'black');
+        
+        $response = getCustomisation('rzp_afd_color');
```
Comment (abdulwahidsharief) [inline tests/phpunit/tests/test-affordabilitywidget.php:83]: @garimaprachi Undefined array key rzp_afd_color

Comment (garimaprachi) [reply tests/phpunit/tests/test-affordabilitywidget.php]: Changed values

File: tests/phpunit/tests/test-affordabilitywidget.php [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+    public function testgetFooterFontSize()
+    {
+        $response = getFooterFontSize();
+
+        $this->assertSame('10', $response);
+    }
+
+    public function testgetCustomisation()
+    {
+        add_option('rzp_afd_theme_color','#8BBFFF');
```
Comment (abdulwahidsharief) [inline tests/phpunit/tests/test-affordabilitywidget.php:81]: @garimaprachi leave space after ,

Comment (garimaprachi) [reply tests/phpunit/tests/test-affordabilitywidget.php]: Done

**Review by abdulwahidsharief — APPROVED** [review]
LGTM

**Pull request merged**